src/org/zaproxy/zap/extension/pscanrulesBeta/CookieLooselyScopedScanner.java

```diff
@@ -85,10 +85,10 @@ private boolean isLooselyScopedCookie(HttpCookie cookie, String host) {
 		assert host != null;
 		
 		String cookieDomain = cookie.getDomain();
-		
+

```

Comment:
Please revert this whitespace change. It could be committed separately if needed.
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines